Makefile

```diff
@@ -19,47 +30,65 @@ ARCHS ?= amd64 arm64 ppc64le s390x
 
 default: clean mod-tidy generate fmt vet-check test build
 
-build: amd64
+promu-prepare: 
+	sed "s/{{.Version}}/$(VERSION)/" .promu.yml >.promu.release.yml
```

Comment:
I guess we should to add `.promu.release.yml` to .gitignore now, right?

Reply:
fixed in force push